src/chat/common/pipes/ws-validation.pipe.ts

```
@@ -0,0 +1,27 @@
+    const object = plainToInstance(metatype, value);
+    const errors = await validate(object);
+    if (errors.length > 0) {
+      const errorMessages = errors
+        .map((err) => Object.values(err.constraints || {}).join(', '))
+        .join('; ');
+      throw new WsException(`Validation failed: ${errorMessages}`);
+    }
+    return object;
+  }
+
+  private toValidate(metatype: Function): boolean {
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)